manifests/run.pp

```diff
@@ -320,7 +320,7 @@
       } else {
         exec { "start ${title} with docker":
           command     => "${docker_command} start ${sanitised_title}",
-          onlyif      => "${docker_command} inspect ${sanitised_title} -f \"{{ .State.Running }}\" | grep false",
+          onlyif      => "${docker_command} inspect ${sanitised_title} -f \"{{ if (.State.Running) }} {{ nil }}{{ end }}\"",
```

Comment:
so it is the same command in both places. the check is different, ie onlyif vs unless. Am i correct in that summation ?

Reply:
was supposed to be a revert from https://github.com/puppetlabs/puppetlabs-docker/pull/451 but I just realised this does not work also. that's why dave changed it to use grep but we don't have that on windows so i'll have to find a command that works on all OSs...